spec.html

```diff
@@ -38063,8 +38087,15 @@ <h1>GeneratorFunction.prototype</h1>
     <!-- es6num="25.2.3" -->
     <emu-clause id="sec-properties-of-the-generatorfunction-prototype-object">
       <h1>Properties of the GeneratorFunction Prototype Object</h1>
-      <p>The GeneratorFunction prototype object is an ordinary object. It is not a function object and does not have an [[ECMAScriptCode]] internal slot or any other of the internal slots listed in <emu-xref href="#table-27"></emu-xref> or <emu-xref href="#table-56"></emu-xref>. In addition to being the value of the prototype property of the %GeneratorFunction% intrinsic, it is the <dfn>%Generator%</dfn> intrinsic (see Figure 2).</p>
-      <p>The value of the [[Prototype]] internal slot of the GeneratorFunction prototype object is the %FunctionPrototype% intrinsic object. The initial value of the [[Extensible]] internal slot of the GeneratorFunction prototype object is *true*.</p>
+      <p>The GeneratorFunction prototype object:</p>
+      <ul>
+        <li>is an ordinary object.</li>
+        <li>is not a function object and does not have an [[ECMAScriptCode]] internal slot or any other of the internal slots listed in <emu-xref href="#table-27"></emu-xref> or <emu-xref href="#table-56"></emu-xref>.</li>
```

Comment:
This seems like it might be a more concise way to express all the other “doesn’t have call, isn’t a function, doesn’t have Construct” instances in the spec